libdnf/conf/OptionBinds.cpp

```diff
@@ -27,40 +27,47 @@
 
 namespace libdnf {
 
-// ========== OptionBinding class ===============
+// ========== OptionBinds::Item class ===============
 
-OptionBinding::OptionBinding(OptionBinds & optBinds, Option & option, const std::string & name,
+OptionBinds::Item::Item(Option & option,
+                             NewStringFunc && newString, GetValueStringFunc && getValueString, bool addValue)
+: option(&option), newStr(std::move(newString)), getValueStr(std::move(getValueString)), addValue(addValue) {}
+
+OptionBinds::Item::Item(Option & option)
+: option(&option) {}
+
+OptionBinds::Item::Item(OptionBinds & optBinds, Option & option, const std::string & name,
                              NewStringFunc && newString, GetValueStringFunc && getValueString,
                              bool addValue)
 : option(&option), newStr(std::move(newString)), getValueStr(std::move(getValueString)), addValue(addValue)
 {
     optBinds.add(name, *this);
 }
 
-OptionBinding::OptionBinding(OptionBinds & optBinds, Option & option, const std::string & name)
+OptionBinds::Item::Item(OptionBinds & optBinds, Option & option, const std::string & name)
 : option(&option)
 {
     optBinds.add(name, *this);
 }
 
-Option::Priority OptionBinding::getPriority() const
+Option::Priority OptionBinds::Item::getPriority() const
 {
     return option->getPriority();
 }
 
-void OptionBinding::newString(Option::Priority priority, const std::string & value)
+void OptionBinds::Item::newString(Option::Priority priority, const std::string & value)
 {
     if (newStr)
         newStr(priority, value);
     else
         option->set(priority, value);
 }
 
-std::string OptionBinding::getValueString() const
+std::string OptionBinds::Item::getValueString() const
 {
-    if (getValueStr) {
+    if (getValueStr)
```

Comment:
Did we agreed on format:
if () {
    asdasdas()
} else {
    adasd()
} 
---- Not important to change